[core]/es_extended/client/modules/actions.lua

```diff
@@ -28,13 +28,19 @@ CreateThread(function()
     while not ESX.PlayerLoaded do Wait(200) end
     while true do
         ESX.SetPlayerData("coords", GetEntityCoords(playerPed))
+        if playerId ~= PlayerId() then
+            playerId = PlayerId()
+
+            ESX.SetPlayerData("id", playerId)
```

Comment:
I don't think this should be in a loop as the player id never changes - it is static. Same goes for the server id.

Reply:
@tomiichx Hi! Ohh yes nice catch it's true thanks.